tools/update-docs.js

```diff
@@ -0,0 +1,20 @@
+const { execSync } = require('child_process')
+const { dirSync } = require('tmp')
+
+const success = async (pluginConfig, { nextRelease, logger }) => {
+  const [major, minor] = nextRelease.version.split('.')
+  const docsVersion = `${major}.${minor}`
+
+  const { name: docsPath } = dirSync()
+
+  const repoOrigin = `https://${process.env.GH_TOKEN}@github.com/karma/karma-runner.github.com.git`
```

Comment:
The URL is pretty cryptic, can you add some comments to explain the magic?

Reply:
Not sure what kind of comment I should put here. Any suggestions?

This is just a regular repository remote one would get if they try to click a Clone button on GitHub. The only added part is GH_TOKEN value in the `userinfo` part of the [URL](https://en.wikipedia.org/wiki/URL), which allows GitHub to authenticate user (`karmarunnerbot` in this case) and authorise the push to the repository.

Reply:
Added the above text as a comment. Hope it makes it more clear.